README.org

```diff
@@ -7,8 +7,34 @@ s-expressions.  S-expressions are defined by the following type:
 type sexp = Atom of string | List of sexp list
 #+end_src
 
-and are rendered as parenthesized lists of strings, e.g., =(This (is
-an) (s expression))=.
+
+** Usage example
+
+In this example we build an Sexp expression that corresponds to
+=(This (is an) (s expression))=, serialize a Sexp expression into a string,
+and then parse a string into an Sexp.
+
+#+begin_src ocaml
+open Sexplib
+open Sexplib.Sexp
```

Comment:
I wouldn't recommend opening Sexplib.Sexp. Better to do something like `Sexp.(List [Atom "foo"; Atom "bar"])`, or even, `Sexp.List [Atom "foo"; Atom "bar"]`, where in the second case you rely on type-directed constructor disambiguation.

Reply:
Thanks for the suggestion. I've done the former.